db/fixtures/miq_searches.yml

```diff
@@ -154,47 +154,47 @@
           value: "true"
     search_type: default
     db: Vm
-- attributes: 
+- attributes:
     name: default_Platform / EC2
     description: Platform / EC2
-    filter: !ruby/object:MiqExpression 
-      exp: 
-        "=": 
+    filter: !ruby/object:MiqExpression
+      exp:
+        "=":
           field: Host-vmm_vendor
-          value: EC2
+          value: EC2 # Is this a valid vendor?
```

Comment:
I think Amazon stuff doesn't exist anymore, so this can probably be removed.  @blomquisg or @gmcculloug  keep me honest here.